admin/resources/js/components/commonMessages.ts

```diff
@@ -0,0 +1,16 @@
+import { defineMessages } from "react-intl";
+
+const messages = defineMessages({
+  loadingTitle: {
+    id: "loadingTitle",
```

Comment:
I think the ids here should start with "common.". As a suggestion for these ids: "common.loading.title" and "common.loading.error"?